QuickFixers/Views/Shared/_SPLayout.cshtml

```diff
@@ -0,0 +1,46 @@
+﻿<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>@ViewBag.Title - Come Get Your Fix!</title>
+    @Styles.Render("~/Content/css")
+    @Scripts.Render("~/bundles/modernizr")
+</head>
+<body>
+    <div class="navbar navbar-inverse navbar-fixed-top">
+        <div class="container">
+            <div class="navbar-header">
+                <button type="button" class="navbar-toggle" data-toggle="collapse" data-target=".navbar-collapse">
+                    <span class="icon-bar"></span>
+                    <span class="icon-bar"></span>
+                    <span class="icon-bar"></span>
+                </button>
+                @Html.ActionLink("QuickFix", "Index", "ServiceProvider", new { area = "" }, new { @class = "navbar-brand" })
+            </div>
+            <div class="navbar-collapse collapse">
+                <ul class="nav navbar-nav">
+                    <li>@Html.ActionLink("Dashboard", "Index", "ServiceProvider")</li>
+                    <li>@Html.ActionLink("Work Schedule", "WorkSchedule", "ServiceProvider")</li>
+                    <li>@Html.ActionLink("Invoices", "Invoices", "ServiceProvider")</li>
+                    <li>@Html.ActionLink("Services Offered", "ServicesOffered", "ServiceProvider")</li>
+                    <li>@Html.ActionLink("Scheduled Services", "ScheduledServices", "ServiceProvider")</li>
+                    <li>@Html.ActionLink("Past Services", "PastServices", "ServiceProvider")</li>
+                    <li>@Html.ActionLink("Log Out", "LogOut", "ServiceProvider")</li>
```

Comment:
Shared/_Layout should already have logout logic in Controllers/HomeController.cs/Logout. Please remove all logout functions and confirm if we can reuse the existing Logout function.

![image](https://user-images.githubusercontent.com/26256313/163688906-1f02163e-12ba-4b5d-90f4-188b3b93183c.png)

![image](https://user-images.githubusercontent.com/26256313/163688924-041b6e51-5f6b-418d-b13d-3412b94fcdbd.png)